### ISSUE TITLE
Full german translation

### ISSUE THREAD

**Issue by OffifialMITX** [vijai1996/screenrecorder#15]
There are 2 strings which aren't translated yet, I've <a href="https://gist.github.com/24dc5da583206539121929a9fbd01fbe">translated them</a> via <a href="https://github.com/LonamiWebs/Stringlate">Stringlate</a>

**Comment by vijai1996** [comment]
I'm still without internet for more than 21 days straight. My ISP's major fibers are cut and they are re-laying the cables. once it is done, I will push an update.

**Comment by OffifialMITX** [comment]
I'm sorry about that and Cooooooool!

Am 5. Januar 2017 05:57:39 MEZ, schrieb Vijai Chander <notifications@github.com>:
>I'm still without internet for more than 21 days straight. My ISP's
>major fibers are cut and they are re-laying the cables. once it is
>done, I will push an update.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/vijai1996/screenrecorder/issues/15#issuecomment-270564141

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.

**Comment by vijai1996** [comment]
Closed. Duplicate of #17
